lib/ansible/plugins/action/dnos10_template.py

```diff
@@ -1,25 +0,0 @@
-# This file is part of Ansible
```

Comment:
Yep, that's what confused me earlier, so I created a new branch and re-did the same git mv's. Looks   like the git doesn't care since the file content are the same. even though I did git mv dnos10_config.py dellos10_config.py, it reflected differently in the status, since the hash value for both the files are same.